LiveSplit.AutoSplitters.xml

```diff
@@ -15358,3 +15358,13 @@
         <Website>https://www.speedrun.com/i_am_fish</Website>
     </AutoSplitter>
 </AutoSplitters>
+<AutoSplitter>
+        <Games>
+            <Game>Scooby-Doo! First Frights
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/kamil2050/ASL-scripts/main/Scooby-Doo!%20First%20Frights%20load%20remover.asl
+        </URLs>
+        <Type>Script</Type>
+        <Description>Load remover for PC version by kamil2050</Description>
+    </AutoSplitter>
```

Comment:
This will not pass checks. You must put the `<AutoSplitter>` element within the root `<AutoSplitters>` element.

```suggestion
    <AutoSplitter>
        <Games>
            <Game>Scooby-Doo! First Frights
        </Games>
        <URLs>
            <URL>https://raw.githubusercontent.com/kamil2050/ASL-scripts/main/Scooby-Doo!%20First%20Frights%20load%20remover.asl</URL>
        </URLs>
        <Type>Script</Type>
        <Description>Load remover for PC version by kamil2050</Description>
    </AutoSplitter>
</AutoSplitters>
```